Makefile

```diff
@@ -16,7 +16,7 @@ generate:
 test:
 	@echo "Running tests on:"; git symbolic-ref HEAD; git rev-parse HEAD
 	go test $(TEST) $(TESTARGS) -timeout=10s
-	@$(MAKE) vet
```

Comment:
The `$(MAKE)` is a built-in make feature that ensures submake calls are called using the same make executable as the original file. This is built-in to make so I think changing this is either wrong or is masking another problem.

Also, this change makes it looks like a program called `MAKE` is being called, which may work on a case-insensitive filesystem but (I think) will break on Linux.